python/plugins/processing/algs/qgis/postgis_utils.py

```diff
@@ -93,12 +93,14 @@ def __init__(self, row):
 class DbError(Exception):
 
     def __init__(self, message, query=None):
-        # Save error. funny that the variables are in utf-8
-        self.message = unicode(message, 'utf-8')
-        self.query = (unicode(query, 'utf-8') if query is not None else None)
+        self.message = unicode(message)
+        self.query = unicode(query)
```

Comment:
Would it work to simply do this?

    self.message = message
    self.query = query

This would preserve a None query in a semantic rather than litteral way